app/controllers/embed/EmbedController.php

```diff
@@ -98,7 +98,7 @@ private function doResponse($title, $playerInfoUri, $registerWatchingUri, $regis
 		$vodPlayStartTime = $kioskMode ? 0 : $this->getVodStartTimeFromUrl();
 		$flushMode = $kioskMode || (!isset($_GET['flush']) || $_GET['flush'] === "1");
 		$showHeading = !$flushMode && (!isset($_GET['showHeading']) || $_GET['showHeading'] === "1");
-		$hideBottomBar = $flushMode;
+		$hideBottomBar = $flushMode || (!$flushMode && (!isset($_GET['hideBottomBar']) || $_GET['hideBottomBar'] === "1"));
```

Comment:
Would be good to keep the default behaviour the same

```suggestion
		$hideBottomBar = $flushMode || $_GET['hideBottomBar'] === "1";
```

Should do it :)